lib/engine/game/base.rb

```diff
@@ -650,6 +649,23 @@ def tile_cost(tile, entity)
         end
       end
 
+      def all_players
+        @players + @bankrupt_players
```

Comment:
i wonder if we shouldn't have all players and just mark playres as bankrupt? thoughts?

Reply:
I think that wound up with a lot of filtering the players for the non-bankrupt ones. This `all_players` approach was less code than my first attempt with a `bankrupt` property.

Reply:
69ce488d